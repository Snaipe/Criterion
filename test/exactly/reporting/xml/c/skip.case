[act]

% skip.c.bin --xml

[assert]

stdout
  -transformed-by ( replace_version | replace_times )
  equals -contents-of expected/skip.txt
