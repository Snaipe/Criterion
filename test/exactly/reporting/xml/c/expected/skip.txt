<?xml version="1.0" encoding="UTF-8"?>
<!-- Tests compiled with Criterion <VERSION> -->
<testsuites name="Criterion Tests" tests="2" failures="0" errors="0" disabled="2">
  <testsuite name="misc" tests="2" failures="0" errors="0" disabled="2" skipped="2" time="<TIME>">
    <testcase name="skipping" assertions="0" status="SKIPPED" time="<TIME>">
      <skipped/>
    </testcase>
    <testcase name="message" assertions="0" status="SKIPPED" time="<TIME>">
      <skipped/>
    </testcase>
  </testsuite>
</testsuites>
